tests/test_units/test_dataio.py

```diff
@@ -82,13 +80,13 @@ def test_config_stratigraphy_empty_entries_alias(globalconfig2, regsurf):
     cfg = deepcopy(globalconfig2)
     cfg["stratigraphy"]["TopVolantis"]["alias"] += [None]
 
-    with pytest.warns(PendingDeprecationWarning, match="'alias' items must be strings"):
-        exp = ExportData(config=cfg, content="depth", name="TopVolantis")
-        metadata = exp.generate_metadata(regsurf)
+    exp = ExportData(config=cfg, content="depth", name="TopVolantis")
+    metadata = exp.generate_metadata(regsurf)
```

Comment:
This test should probably xfail at least. I am not up to date -- why is stratigraphic_alias skipped here?

Reply:
I have no idea, that current test looks like a placeholder to me. So i made i complete IMO, and then marked it as xfail as it will fail until the stratigraphic_alias is fully implemented.